parquet-variant/src/builder.rs

```diff
@@ -114,11 +113,11 @@ fn make_room_for_header(buffer: &mut Vec<u8>, start_pos: usize, header_size: usi
 /// };
 /// assert_eq!(
 ///   variant_object.field_by_name("first_name").unwrap(),
-///   Some(Variant::ShortString("Jiaying"))
+///   Some(Variant::ShortString(ShortString::try_new("Jiaying").unwrap()))
```

Comment:
These examples might be nicer with the `From` impl (but we can do that as a follow on PR)

```suggestion
///   Some(Variant::from("Jiaying"))
```